yb-voyager/src/tgtdb/oracle.go

```diff
@@ -587,7 +589,18 @@ func (tdb *TargetOracleDB) InitConnPool() error {
 	return nil
 }
 
-func (tdb *TargetOracleDB) GetDebeziumValueConverterSuite() map[string]ConverterFn {
+func (tdb *TargetOracleDB) GetDebeziumValueConverterSuite() map[string]tgtdbsuite.ConverterFn {
+	oraValueConverterSuite := tgtdbsuite.OraValueConverterSuite
+	for _, i := range []int{1, 2, 3, 4, 5, 6, 7, 8, 9} {
```

Comment:
is 1-9 the precision of the milliseconds? Can you just add documentation in comments?

Reply:
yes, its precision, will add comment

Reply:
oops my bad.